### ISSUE TITLE
Resource Not Present in Version Control

### ISSUE THREAD

**Issue by onlinejudge95** [ShashwatMishra/Twiiter-Bot#1]
Please explain the following [import process](https://github.com/ShashwatMishra/Twiiter-Bot/blob/4a2528a0f5bcb6a7b92d813d366a785d18cf86f0/Main#L12). 

I created a Fork for a Pull Request to refactor the codebase, but I am unable to replicate a successful build even after installing all dependencies **(Which by the way had no version info so, I am not sure that I would ever be able to fully replicate a same build)**. I am unable to launch the web server due to this, so please provide timely measure.

**Comment by ShashwatMishra** [comment]
onlinejudge95 , did u made a developer account on twitter .
This application will work when you have consumer key and consumer secret key .

**Comment by onlinejudge95** [comment]
Yes @ShashwatMishra I have a developer account and regularly syncing my api keys from there.

Following through the book on my own. Deleted the fork and made my own repo about it, you can close the issue if you want.

Also it would be better to document the above step in a README.md
